website/content/api-docs/secret/transit.mdx

```diff
@@ -79,6 +81,15 @@ values set here cannot be changed after key creation.
 
   ~> **Note**: When key type is `managed_key`, either the `managed_key_name` or
      `managed_key_id` parameter must be specified.
+
+  ~> **Note**: When key type is `ml-dsa`, `parameter_set` must be specified.
```

Comment:
Wonder if we should have a default parameter_set?

Reply:
IMO that wouldn't work well once we have SLH-DSA, since they'll both use `parameter_set`, and the values don't have any overlap